.idea/compiler.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The idea here would be to ignore "IDE" files, like .idea/ folder or .iml. Take a look at https://www.gitignore.io/